.github/workflows/tests_secondary.yml

```diff
@@ -159,14 +160,14 @@ jobs:
         PLAYWRIGHT_SKIP_BROWSER_DOWNLOAD: 1
     - run: npm run build
     - run: npx playwright install --with-deps ${{ matrix.browser }} chromium
-    - run: xvfb-run --auto-servernum --server-args="-screen 0 1280x960x24" -- npm run test -- --project=${{ matrix.browser }}
-      if: ${{ always() }}
-      env:
-        HEADFUL: 1
+    - run: xvfb-run --auto-servernum --server-args="-screen 0 1280x960x24" -- npm run test -- --project=${{ matrix.browser }} --headed
+      if: always() && matrix.os == 'ubuntu-latest'
+    - run: npm run test -- --project=${{ matrix.browser }} --headed
+      if: always() && matrix.os != 'ubuntu-latest'
     - run: ./utils/upload_flakiness_dashboard.sh ./test-results/report.json
-      if: always()
+      if: always() && matrix.os == 'ubuntu-latest'
```

Comment:
Do this for every OS to get results on the dashboard.

Reply:
I don't know how to, Lusha can make it happen!